packages/components/psammead-caption/src/index.stories.jsx

```diff
@@ -11,9 +11,12 @@ storiesOf('Components|Caption', module)
   .addDecorator(withKnobs)
   .add(
     'default',
-    inputProvider([{ name: 'Caption' }], ([captionText], script) => (
-      <Caption script={script}>{captionText}</Caption>
-    )),
+    inputProvider(
```

Comment:
This is the old API for this package, this is on `@bbc/psammead-storybook-helpers` `"^2.1.1"`, if you link them locally it uses v3, which is a breaking change.

We will need to bump them all to `3.1.0` to fix this API issue :P

Reply:
We use `--force-local` which overrides semver breaking change

Reply:
@dr3 ah, if v2 used that API, that would explain why they worked when not linked

So just to be clear - my code changes here are correct, I also just need to bump the package.json of all the changed components to use the latest version of `storybook-helpers`?

Reply:
Yep :) We should raise an issue to review our use of `--force-local` too, to avoid issues with breaking changes in future. I think we had issues in the past where it wouldnt link packages with different minor versions, but that might be fine now